Hangman/Hangman/main.swift

```diff
@@ -8,5 +8,168 @@
+// option command left
+// to unfold is in editor
+
+    // Instructions
+print("WELCOME tooo Hangman, a game that is much more tedious and stressfull than your middle school spelling bee because one wrong move and someone's gonna end up dead(figruatively of course).")
+print()
+
+print("Please be mindful that you only have 6 times to miss a letter before you are hanged... Also we only want letters NOT numbers. ")
+
+print()
+
+print("Your word has been chosen... Now it time for you to guess the word.. good luck.")
+print()
+
+    //This is the word that the computer chose.
+var randomWord = allTheWords.randomElement() ?? ""
+    //This counts the lenght of the word
+var randomWordCount = randomWord.count
+
+    //prints the word so I can see it and I can remember will delete before final submission
+print(randomWord)
+
+    // is the place holder for the amount of letters in a word
+var placeHolder = "_ "
+    // this equates amount of letters in the chosen word to the amount of place holders
+//if randomWordCount != placeHolder.count {
+//    //This makes the place holder repeat itself to the amount that of word count
+//let amountOfPlaceHolder = String(repeating: "_ ", count: randomWordCount)
+//    print(amountOfPlaceHolder,terminator:"")
+//}
+
+//for loop that add "_" to amountOfPlaceHolder depending on the count of the random word
+// if statement is check something
+var amountOfPlaceHolder = [Character]()
+for _ in 1...randomWordCount { // loops the characters through the word..
+    // starts from 1 to amount of letters
+    amountOfPlaceHolder.append("_")
+}
+
+    // This prints the place holders in one line
+
+    // Establishes the variable for the amount of guesses the user has.
+    //amount of lives used
+var numberOfAttempts = 6
+print()
+
+print("please enter in a letter you would like to guess")
+print()
+
+//need to make sure user cannnot enter more than 1 letter or a number??
+
+
+//The user enters a letter for the random word and allows for it to be a upper case letter.
+//if userLetter = Int
+
+// create a repeat while loop that allows for the computer to check if the letter inputed is correct
+
+//var letter = userLetter
+var condition = true // this is only to verify the while which should be either true or false but to simplifiy it you use a variable that holds the place of true or false
```

Comment:
Would this have been the line that determined whether the user wanted to play again? This would have worked if you added a line of code that included a conditional statement along with a readLine asking the user if they would like to play again to select "yes" or "no" and toggle the Bool based on their response.